src/SetupApi/SetupApi+OpenFlags.cs

```diff
@@ -0,0 +1,40 @@
+﻿// Copyright © .NET Foundation and Contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace PInvoke
+{
+    using System;
+    using System.Diagnostics.CodeAnalysis;
+
+    /// <content>
+    /// Contains the <see cref="GetClassDevsFlags"/> nested type.
+    /// </content>
+    public partial class SetupApi
+    {
+        /// <summary>
+        /// Controls how the device information element is opened.
+        /// </summary>
+        [Flags]
+        public enum OpenFlags : uint
```

Comment:
Unless this enum shows up in more than one method, I believe it should be called `SetupDiOpenDeviceInfoFlags` to match our [naming guidelines](https://github.com/dotnet/pinvoke/blob/master/CONTRIBUTING.md#naming).